### DISCUSSION TITLE
#16. VATZ biweekly meeting at `2022-08-31 08:00:00 -06:00:00`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#234] [General]
### 1. Overall
![image](https://user-images.githubusercontent.com/6308023/185273883-319ab182-ba4c-4a21-98bd-2c61c120912b.png)

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
14|  13/23 | 43
15|  4/16 | 20

> Thanks for your hard work for 2 weeks.
> The target progress rate for each sprint is 40% to 55%. 
> YES!!

---

- Finished
  - #233
  - #230 
  - #224
  
- Move to Sprint 16
   - #222 
   - #85 
   - #87 
   - #119 
   - #120 
   - #121 
   - #122 
   - #146 
   - #174 
   - #175 
   - #187 
   - #197 
   - #202 
   - #225 
   - #226 
   - #228  
---
### discussion
   - #225
   - #120

**Comment by rootwarp** [comment]
# Joonkyo

- https://github.com/dsrvlabs/vatz/issues/178
- https://github.com/dsrvlabs/vatz-proto/issues/9
- https://github.com/dsrvlabs/vatz/issues/107
  - https://github.com/dsrvlabs/vatz/issues/220
  - https://github.com/dsrvlabs/vatz/issues/222

**Reply by xellos00** [reply]
Grpc -> https transferreing as APIs

**Comment by xellos00** [comment]
# 동요

- #233
- #120
- #186 
   - #229
- #225
- #238

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/226

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/schellingpoint-project/issues/10

**Reply by xellos00** [reply]
@skonhwang 
Please, register new repo!!

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/196

**Comment by heejin-github** [comment]
## 희진
- #236 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/1
  - https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/1

**Comment by xellos00** [comment]
@dsrvlabs/validator 
모두 2주 동안 고생하셨습니다. 👍🏽
